src/PrestaShopBundle/Form/Admin/Sell/Product/ShortcutsType.php

```diff
@@ -41,18 +43,52 @@ class ShortcutsType extends TranslatorAwareType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
+            ->add('reference', UnavailableType::class, [
+                'label' => $this->trans('Reference', 'Admin.Catalog.Feature'),
+                'label_tag_name' => 'h2',
+                'label_attr' => [
+                    'popover' => $this->trans('Your reference code for this product. Allowed special characters: .-_#.', 'Admin.Catalog.Help'),
+                ],
+            ])
             ->add('stock', StockShortcutType::class, [
                 'label' => $this->trans('Quantity', 'Admin.Catalog.Feature'),
-                'help' => $this->trans('How many products should be available for sale?', 'Admin.Catalog.Help'),
+                'label_tag_name' => 'h2',
+                'label_attr' => [
+                    'popover' => $this->trans('How many products should be available for sale?', 'Admin.Catalog.Help'),
+                ],
+                'required' => false,
                 'target_tab' => 'stock-tab',
                 'target_tab_name' => $this->trans('Quantity', 'Admin.Catalog.Feature'),
             ])
             ->add('price', PriceShortcutType::class, [
                 'label' => $this->trans('Retail price', 'Admin.Catalog.Feature'),
-                'help' => $this->trans('This is the retail price at which you intend to sell this product to your customers. The tax included price will change according to the tax rule you select.', 'Admin.Catalog.Help'),
+                'label_tag_name' => 'h2',
+                'label_attr' => [
+                    'popover' => $this->trans('This is the retail price at which you intend to sell this product to your customers. The tax included price will change according to the tax rule you select.', 'Admin.Catalog.Help'),
```

Comment:
@jolelievre We cannot find this string anywhere on the page. 🤔  Do you know where it is supposed to appear? + it is very similar to this string: 
![image](https://user-images.githubusercontent.com/70583503/116377589-9ef99800-a811-11eb-8b04-493269933e28.png)